### PR TITLE
Use common chunker for archive handler

### PR DESCRIPTION
**Pull request by ahrav**
### Description:
Leverage the common chunker within the archive handler. This should also make sure the chunks include peek data.

### Checklist:
* [ ] Tests passing (`make test-community`)?
* [ ] Lint passing (`make lint` this requires [golangci-lint](https://golangci-lint.run/usage/install/#local-installation))?

### CODE REVIEW

**Review by rosecodym — APPROVED** [review]
i don't really know how the archive stuff works so take this approval with a grain of salt

**Comment by ahrav** [comment]
> i don't really know how the archive stuff works so take this approval with a grain of salt

I'll take that approval as an initial okay that the code looks not too bad. I'll wait for another review to make sure using the sliding window w/ overlap chunker in the archive handler looks okay.

**Pull request merged**
